src/ServicePulse.Host/ServicePulse.Host.csproj

```diff
@@ -101,6 +101,7 @@
     <Content Include="app\js\directives\ui.particular.failedMessageTabs.tpl.html" />
     <Content Include="app\js\directives\ui.particular.nodata.js" />
     <Content Include="app\js\directives\ui.particular.nodata.tpl.html" />
+    <Content Include="app\js\services\factory.rxjs.js" />
```

Comment:
tbh I don't see any value in having a dedicated file for this, especially since I'm not aware of any minification steps.